Makefile

```diff
@@ -550,7 +550,7 @@ e2e-image: ## Build the image for e2e tests
 
 .PHONY: e2e-image-push
 e2e-image-push: e2e-image ## Push the image for e2e tests
-	TAG=v0.0.1 CONTROLLER_IMAGE_NAME=turtles-e2e $(MAKE) docker-push
+	TAG=v0.0.1 CONTROLLER_IMAGE_NAME=turtles-e2e $(MAKE) docker-build-and-push
```

Comment:
I believe we can clean up/get rid of possibly `docker-build` target, since in this case we are calling `e2e-image` which already calls `docker-build` and `docker-build-and-push` repeats the same with push addition. However, it it is not a blocker for this PR